javascript/javascript1/week3/smart-easy-back-to-the-basics.js

```diff
@@ -0,0 +1,65 @@
+const notes = [];
+var i = 0;
+function saveNote(content, id) {
+  notes.push({});
+  for (; i < notes.length; i++) {
+    notes[i].content = content;
+    notes[i].id = id;
+  }
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Run dishwasher", 3);
+saveNote("Cook meal", 4);
+
+console.log(notes);
+
+function getNote(id) {
+  if (typeof id == "number") {
+    for (i = 0; i < notes.length; i++) {
+      if (notes[i].id === id) {
+        return notes[i];
+      } else {
+        const error1 = "Error: This id does not exist.";
+        return error1;
+      }
+    }
+  } else {
+    const error2 = "Error: Please enter a number.";
+    return error2;
+  }
+}
+
+const firstNote = getNote(6);
+console.log(firstNote);
+
+function logOutNotesFormatted() {
+  for (var i = 0; i < notes.length; i++) {
+    return (
```

Comment:
this thing works only for the first note. 
because loop is interrupted at the very first iteration